rpm/fs/etc/init.d/mad

```diff
@@ -0,0 +1,167 @@
+#!/bin/sh
+#
+# mad Metrics aggregation daemon.
+#
+# chkconfig: - 60 40
+# description: MAD is an aggregation daemon that computes local \
+#   statistics on data and forwards them to upstream aggregation \
+#   services
+# processname: mad
+# config: /opt/mad/config/config.json
+# pidfile: /var/run/mad/mad.pid
+
+# Source function library.
+. /etc/rc.d/init.d/functions
+
+user="mad"
+prog="mad"
+dir="/opt/mad"
+config="/opt/mad/config/config.json"
+exec="/opt/mad/bin/mad"
+pid_file="/var/run/mad/mad.pid"
+export JAVA_HOME="$(dirname $(dirname $(readlink -f $(which java))))"
+
+[ -e /etc/sysconfig/$prog ] && . /etc/sysconfig/$prog
+
+lockfile=/var/lock/subsys/$prog
+
+get_pid() {
+    if [ -f "$pid_file" ]; then
+        cat "$pid_file" 2> /dev/null
+    fi
+    echo ""
+}
+
+is_running() {
+    [ ! -f "$pid_file" ] && return 1
+    kill -0 `get_pid` &> /dev/null && return 0 || return 1
+}
+
+start() {
+    [ -f $config ] || exit 6
+    echo -n $"Starting $prog: "
+    if is_running; then
+        warning "$name: already running"
+    else
+        cd "$dir"
+        su "$user" -s /bin/bash -c "nice $exec $config >> $dir/system.out 2>&1 & echo \$! > $pid_file"
```

Comment:
Does this need to be bash specifically instead of say sh; I'm thinking whether we then need to declare it as a dependency for the rpm -- and if so, is it necessary to be bash instead of sh?